flask/todo_app/modules/todo/todo_views.py

```diff
@@ -1,10 +1,13 @@
-from flask import Blueprint, render_template
+from ct_core_api.common import app_utils
+
+from flask import Blueprint
 from flask_login import login_required
 
 bp = Blueprint('view.todo', __name__)
 
 
 @bp.route("/")
+@app_utils.template()
```

Comment:
It renders the template with the name of the function + ".html" by default.
If the function returns a dict, it'll be used as the template vars.
Just a convenience decorator.